meetings/2021-12/dec-14.md

```diff
@@ -0,0 +1,932 @@
+
+KG: Yes, that's an excellent point.
+
+## ECMA-402 Status
+
+Presenter: Ujjwal Sharma (USA)
+
+USA: In the interest of time, I'd really quickly make the statement that there is no update and so there's no new normative pull request that we need consensus on, which means that we're free for ecma 402. Thank you.
+
+## ECMA-404 Status
+
+RPR: Anything on Ecma 404, Chip?
+
+CM: It's exactly what you expect. All is well, all is stable. Nothing changes. That's the way it's supposed to be.
+
+## Test262 Status
+
+Presenter: Rick Waldron (RW)
+
+RW: Regular additions and updates. Frequency of contribution. It's all exactly as I would expect it to be for a large-scale repository full of conformance tests. Stage 3 stuff is doing great. If anybody has any questions, hit me up.
+
+## Updates from the CoC committee
+
+JHD: I don't think there's really any updates right now.
+
+## Add import.meta[Symbol.toStringTag]
+
+Presenter: Jordan Harband (JHD)
+
+- [PR](https://github.com/tc39/ecma262/pull/2106)
+
+JHD: So, the committee has previously decided essentially that name space objects like reflection and atomics should have a @@toStringTag for debuggability.
+
+Only really, what is the proposed content of the two string tag? Oh, in this pull request. It is the string import meta in Pascal case.
+
+MM:  In what case Pascal-like capital?
+
+JHD: Yes, capital I.
+
+MM: Got it. it. Thank you.
+
+JHD: Yeah, and it could be anything the exact content of the string is I think pretty byte shuttle. The thing that we need consensus on most. Most importantly here is, should we add the property or not? The arguments for would be debug ability and you know, it's a built-in object that holds stuff like reflective Amendment and you know math and atomics  and so on. The arguments against it would be that, it's not like a globally available one where the same instance is always everywhere, each module gets their own so, Bradley commented that PR year and a half ago for some reasons why we may be shouldn't do it. So, the analog here, there's three bullet points. The fourth bullet point here to me, is sort of contradicted by the fact that module namespace objects already have own symbol that to string tag properties. So even though every module has can be imported as its own module name, space object, that will have a symbol to string had property. So for me, the that can import that matter are conceptually similar. There's no like, you know, one per module and The hold on Jordan get have bugged.
```

Comment:
```suggestion
JHD: Yeah, and it could be anything the exact content of the string is I think pretty bikesheddable. The thing that we need consensus on most importantly here is, should we add the property or not? The arguments "for" would be debuggability. It's a built-in object that holds stuff like Reflect, Math, Atomics, and so on. The arguments "against" would be that, it's not like a globally available one where the same instance is always everywhere, each module gets their own. Bradley commented on that PR a year and a half ago with some reasons why we maybe shouldn't do it - there's three bullet points. The fourth bullet point here to me, is sort of contradicted by the fact that module namespace objects already have own `Symbol.toStringTag` properties. So even though every module can be imported as its own module namespace object, that will have a `Symbol.toStringTag` property. So for me, that and `import.meta` are conceptually similar. There's not one per module.
```